index.html

```diff
@@ -0,0 +1,289 @@
+</head>
+
+<body>
+	<div class="page">
+		<div class="container">
+			<header>
+				<ul>
+					<li><a class="myLogo font-style" href="">My Logo</a> </li>
+					<nav class="font-style">
+						<li><a class="toggler-icon"><img class="toggler-img" src="images/humburger.png"
+									alt="humberger-icon"></a>
+						</li>
+					</nav>
+
+				</ul>
+
+
+			</header>
+			<div class="main">
+				<section class="home-page">
+					<section class="intro-section">
+						<h1 class="main_heading font-style">I'm Fentahun <br> Glad to see you!</h1>
+						<p class="description font-style">
+							I’m a software developer! I can help you build a product , feature or
+							website Look through some of my work and experience! If you like what
+							you see and have a project you need coded, don’t hestiate to contact
+							me.
+						</p>
+					</section>
+
+					<section>
+						<div class="connect-section">
+							<h4 class="connect font-style">LET'S CONNECT</h4>
+							<ul class="socials">
+								<li>
+									<a class="social_medias twitter" href="https://twitter.com/FentahunAgmas">
+										<img src="images/Vector (1).svg" alt="twitter logo">
+									</a>
+								</li>
+
+								<li>
+									<a class="social_medias"
+										href="https://www.linkedin.com/in/fentahun-agmas-5a01aa162/">
+										<img src="images/Linkedin icon.svg" alt="Linkedin icon">
+									</a>
+								</li>
+								<li>
+									<a class="social_medias" href="">
+										<img src="images/mediumLogo.png" alt="Medium icon" class="medium" />
+									</a>
+								</li>
+								<li>
+									<a class="social_medias" href="https://github.com/Fentahunn">
+										<img src="images/githublogo.png" alt="Github icon" />
+									</a>
+								</li>
+								<li>
+									<a href="#"><img src="images/Vector (2).svg" alt="two fingers"></a>
+								</li>
+							</ul>
+						</div>
+
+					</section>
+				</section>
+			</div>
+
+		</div>
+
+
+		<!-- project -->
+		<section id="portfolio" class="cards grid-container">
+			<div class="first-card card sec-div">
+				<img class="snapshoot-style" src="images/Snapshoot Portfolio.svg" alt="Snapshoot portfolio image">
+				<div class="sec-div">
+					<h2 class="card-header font-style on-small">
+						Tonic
+					</h2>
+					<div class="sec-div card-description">
+						<ul class="font-style">
+							<li class="card-list-title on-small"> CANOPY</li>
+							<li class="Job-title_and_year on-small">
+								Back End Dev
+							</li>
+							<li class="Job-title_and_year">
+								2023
+							</li>
+						</ul>
+					</div>
+					<p class="notice font-style">
+						A daily selection of privately personalized reads; no accounts or sign-ups required.
+					</p>
+					<ul class="programming-languages font-style">
+						<li>html</li>
+						<li>css</li>
+						<li>javaScript</li>
+					</ul>
+					<button id="0" class="project-button button" type="button">See Project</button>
+				</div>
+				</di>
+			</div>

```

Comment:
According to the Figma design, I suggest you make an update to your design by improving some padding and margin.

> ## The Figma Design
>
><img width="323" alt="Screenshot 2023-06-02 at 8 31 08 PM" src="https://github.com/Fentahunn/finish_mobile_version/assets/91237525/69190126-821b-4a7d-a4c0-a1beed662a22">
>
> ## Your Design
>
><img width="301" alt="Screenshot 2023-06-02 at 8 31 54 PM" src="https://github.com/Fentahunn/finish_mobile_version/assets/91237525/1cb76e37-ca63-4536-8278-76e6b6cb0978">